bocadillo/error_handlers.py

```diff
@@ -46,6 +48,7 @@
     ```
     """
     res.status_code = exc.status_code
+    res.headers = exc.headers
```

Comment:
I think some of the tests failing in CI come from this line. We shouldn’t set response headers if `exc.headers is None`. :)

Reply:
oh duh its optional, ill change that. well what about adding it to __slots__?

Reply:
__slots__ = (status, detail, headers ) -> ?

Reply:
We can certainly add it, yes.